src/examples/assemblies.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: 2025 Tsolo.io
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from cycax.cycad import Assembly, Cuboid, Print3D, SheetMetal
+from cycax.cycad.engines.assembly_build123d import AssemblyBuild123d
+from cycax.cycad.engines.part_build123d import PartEngineBuild123d
+
+LENGTH = 100
+
+
+class Bracket(Print3D):
+    """This class represents a special bracket to hold a device in place."""
+
+    def __init__(self):
+        super().__init__(part_no="bracket", x_size=LENGTH, y_size=50, z_size=15)
+
+    def definition(self):
+        """Define the bracket."""
+        side = self.front
+        x = 10
+        while x < LENGTH:
+            pos = (x, self.z_size / 2)
+            side.hole(pos=pos, diameter=3.2, depth=2)
+            side.hole(pos=pos, diameter=2.9, depth=10)
+            side.hole(pos=pos, diameter=3.2, external_subtract=True)  # Through everything
+            x += 10
+
+
+class Device(Cuboid):
+    """Create the device."""
+
+    def __init__(self):
+        super().__init__(part_no="sxdevice", x_size=LENGTH, y_size=80, z_size=20)
+
+    def definition(self):
+        """Define the device."""
+        for side in (self.left, self.right):
+            for x in range(10, 50, 90):
```

Comment:
@HelenForbes in this release something suspicious is happening. Maybe me, but can you have a look. 

When I build this I dont get the 3 holes on the sides, the holes are not even in the JSON.

All tests passed, When I get a moment I'll write a test for this scenario.